content/00_Foreword.md

```diff
@@ -1,185 +1,105 @@
-					style="width:120px;height:240px;margin: 10px" 
-					scrolling="no" marginwidth="0" marginheight="0" frameborder="0"></iframe>
-
-		</div>
-		<div style="display:inline-block; vertical-align: top;  margin: 10px; width: 140px; font-size: 11px; text-align: center">
-			<!--C# Linq pocket reference Amazon.co.uk-->
-			<iframe src="http://rcm-uk.amazon.co.uk/e/cm?t=int0b-21&amp;o=2&amp;p=8&amp;l=as1&amp;asins=0596519249&amp;ref=qf_sp_asin_til&amp;fc1=000000&amp;IS2=1&amp;lt1=_blank&amp;m=amazon&amp;lc1=0000FF&amp;bc1=000000&amp;bg1=FFFFFF&amp;f=ifr" 
-					style="width:120px;height:240px;margin: 10px" 
-					scrolling="no" marginwidth="0" marginheight="0" frameborder="0"></iframe>
-		</div>
-
-		<div style="display:inline-block; vertical-align: top; margin: 10px; width: 140px; font-size: 11px; text-align: center">
-			<!--CLR via C# v4 Amazon.co.uk-->
-			<iframe src="http://rcm-uk.amazon.co.uk/e/cm?t=int0b-21&amp;o=2&amp;p=8&amp;l=as1&amp;asins=B00AA36R4U&amp;ref=qf_sp_asin_til&amp;fc1=000000&amp;IS2=1&amp;lt1=_blank&amp;m=amazon&amp;lc1=0000FF&amp;bc1=000000&amp;bg1=FFFFFF&amp;f=ifr" 
-					style="width:120px;height:240px;margin: 10px" 
-					scrolling="no" marginwidth="0" marginheight="0" frameborder="0"></iframe>
-
-		</div>
-		<div style="display:inline-block; vertical-align: top; margin: 10px; width: 140px; font-size: 11px; text-align: center">
-			<!--Real-world functional programming Amazon.co.uk-->
-			<iframe src="http://rcm-uk.amazon.co.uk/e/cm?t=int0b-21&amp;o=2&amp;p=8&amp;l=as1&amp;asins=1933988924&amp;ref=qf_sp_asin_til&amp;fc1=000000&amp;IS2=1&amp;lt1=_blank&amp;m=amazon&amp;lc1=0000FF&amp;bc1=000000&amp;bg1=FFFFFF&amp;f=ifr" 
-					style="width:120px;height:240px;margin: 10px" 
-					scrolling="no" marginwidth="0" marginheight="0" frameborder="0"></iframe>
-
-		</div>           
-	</div>
</div>
-
+---
+title : Foreword
+---
+
+# Introduction to Rx
+By Ian Griffiths and Lee Campbell
+   
+---
+
+Reactive programming is not a new concept. Any kind of user interface development
+necessary involves code that responds to events. Languages like Smalltalk, Delphi
+and the .NET languages have popularized reactive or event-driven programming paradigms.
+Architectural patterns such as CEP (Complex Event Processing), and
+CQRS (Command Query Responsibility Segregation) have events as a fundamental part
+of their makeup. Reactive programming is a useful concept in any program that has
+to deal with things happening.
+
+> Reactive programming is a useful concept in any program that has
+to deal with things happening.
+
+The event driven paradigm allows for code to be invoked without the need for breaking
+encapsulation or applying expensive polling techniques. There are many common ways to implement this, including
+the Observer pattern, events exposed directly in the language (e.g. C#) or
```

Comment:
Would this benefit from a link, too? (Including the C# events - it could link to the MS documentation)